components/Avatar/avatar.css

```diff
@@ -0,0 +1,133 @@
+ @import "/reset.css";
+/* 
+CSS FOR AVTAR
+*/
+.box{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 80px;
+    justify-content: center;
+    gap: 60px;
+}
+.avtar img{
+    width: 3rem;
+    height: 3rem;
+    font-size: 1rem;
+    
+}
+.avtar-flex{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 40px;
+}
+.avtar-txt{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: grey;
+    background-color: rgb(250, 250, 250);
+}
+.avtar-lg-size{
+    width: 10rem;
+    height: 10rem;
+    font-size: 3rem;
+    border-radius: 50%;
+    border: 0px solid var(--avtar-shadow);
+    box-shadow: 0px 4px 10px var(--avtar-shadow);
+}
+.avtar-md-size{
+    width: 6.25rem;
+    height: 6.25rem;
+    font-size: 2rem;
+}
+.avtar-sm-size{
+    width: 3rem;
+    height: 3rem;
+    font-size: 1rem;
+}
+.avtar-xs-size{
+    width: 2rem;
+    height: 2rem;
+    font-size: 0.8rem;  
+}
+.round-border{
+    border-radius: 50%;
+    border: 0px solid var(--avtar-shadow);
+    box-shadow: 0px 4px 10px var(--avtar-shadow);
+}
+.square-border{
+
+    border: 0px solid var(--avtar-shadow);
+    box-shadow: 0px 4px 10px var(--avtar-shadow);
+    border-radius: 10%;
+}
+
+/* 
+CSS FOR AVTAR WITH BADGES
+*/
+.container{
+    display: flex;
+    align-items: center;
+    gap: 30px;
+}
+.avtar-box{
+    position: relative;
+    box-sizing: border-box;
+    width: fit-content;
+}
+.avtar-status{
+        height: 1.25rem;
+        min-width: 1.25rem;
+        padding: 2px;
+        border-radius: 50%;
+        border: 2px solid rgb(3, 3, 3);
```

Comment:
try using color with variables and give it in root.

Reply:
ok done